### DISCUSSION TITLE
Erro ao validar a qntd de caracteres no angular

### DISCUSSION THREAD

**Discussion by EIBANHES** [balta-io/forum#97] [Tira Dúvidas] [answered]
Boa noite professor, sou um novo aluno aqui, estou fazendo introdução ao angular, portanto tive esse impecílio:
src/app/app.component.html:29:33 - error TS4111: Property 'title' comes from an index signature, so it must be accessed with ['title'].
como resolvo??

Att. Emanuel Barros Ibanhes

**Comment by EIBANHES** [comment, marked as answer]
Consegui resolver, na versão do angular 14 invés de form.controls.title.invalid passou para form.controls['title'].invalid, posso tamém mudar o tsconfig.json de "noPropertyAccessFromIndexSignature": true, para "noPropertyAccessFromIndexSignature": false,

**Reply by andrebaltieri** [reply]
Sensacional! Obrigado por compartilhar <3
